### PR TITLE
[WIP]Lightwave compatibility

### PR DESCRIPTION
**Pull request by frlnx**
Lightwave ouputs object files with two instructions, unknown to PyWavefront
Object: g for group
Material: Tf for transmission filter

The first is loosely implemented, the second is simply ignored as it is hard to do in OpenGL

### CODE REVIEW

File: pywavefront/material.py [hunk 1]
```diff
@@ -157,3 +158,8 @@ def parse_Tr(self, args):
     def parse_illum(self, args):
         # unimplemented
         return
+
+    def parse_Tf(self, args):
+        if args != ['1', '1', '1']:
+            stderr.write("Texture Tf (transmission filter) not supported")
```
Comment (greenmoss) [inline pywavefront/material.py]: Should this be log.warn?

File: pywavefront/material.py [hunk 2]
```diff
@@ -32,6 +32,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # ----------------------------------------------------------------------------
 
+from sys import stderr
```
Comment (greenmoss) [inline pywavefront/material.py]: How about importing logger here? Gives the module importer more flexibility in handling the error.

**Comment by frlnx** [comment]
I can add logging. No problem. Got some more compatibility issues and general bugfixes fixed too. I guess I should make this merge request a work in progress.

**Comment by greenmoss** [comment]
@frlnx how are you doing? 😄 

I just merged another PR, and noticed this one is still open.

**Comment by einarf** [comment]
The obj and mat files in this PR actually do parse and render correctly with the new parser.

screen shot 2018-06-14 at 20 15 07

**Comment by greenmoss** [comment]
Thanks for checking @einarf !

@frlnx let us know if you have a different use case that this doesn't cover and we can revisit.

Closing as fixed.

**Pull request closed without merging**